lib/pleaserun/detector.rb

```diff
@@ -79,8 +79,8 @@ def self.detect_ohai
   def self.detect_facter
     require "facter"
 
-    platform = Facter.fact["operatingsystem"]
-    version = Facter.fact["operatingsystemrelease"]
+    platform = Facter.value(:operatingsystem)
```

Comment:
hmm, this may break compatibility with older facter versions, but I'm ok wiating until someone needs older support. +1 on this change